include/vast/Conversion/Passes.td

```diff
@@ -125,6 +125,16 @@ def HLToLLVars : Pass<"vast-hl-to-ll-vars", "mlir::ModuleOp"> {
   let dependentDialects = ["mlir::LLVM::LLVMDialect", "vast::ll::LowLevelDialect"];
 }
 
+def HLToLLFunc : Pass<"vast-hl-to-ll-func", "mlir::ModuleOp"> {
+  let summary = "Convert hl functions into ll versions.";
+  let description = [{
+    This pass is still a work in progress.
+  }];
+
+  let constructor = "vast::createHLToLLFuncPass()";
+  let dependentDialects = ["mlir::LLVM::LLVMDialect", "vast::ll::LowLevelDialect"];
```

Comment:
probably also Core dialect?

Reply:
I don't think so actually. Afaik it should contain dialects from which the pass creates new entities. I should probably remove the llvm dialect that I copied form somewhere else though.